rest_framework/schemas/openapi.py

```diff
@@ -471,13 +470,15 @@ def map_field(self, field):
         if isinstance(field, serializers.FloatField):
             content = {
                 'type': 'number',
+                'format': 'float'
             }
             self._map_min_max(field, content)
             return content
 
         if isinstance(field, serializers.IntegerField):
             content = {
-                'type': 'integer'
+                'type': 'integer',
+                'format': 'int64'
```

Comment:
This seems wrong, are we sure that integer fields are always 64 bits? I have been assuming they are 32 bits on most databases.